src/audio/host-zephyr.c

```diff
@@ -213,6 +214,13 @@ static int host_copy_one_shot(struct host_data *hd, struct comp_dev *dev, copy_c
 		return ret;
 	}
 
+	/* SRC/DEST addresses have changed so the DMAC needs
```

Comment:
What is DMAC?

Also, this commit and a few others (IMO) are these related with native drivers or with the switch from DMA to Timer domain?

Reply:
@iuliana-prodan "DMA Controller"

Reply:
everything here is related to the native drivers switch. Initially these were all part of the PR that switched 93 to native drivers but I've moved them here so that we don't get blocked by the Zephyr hash bump.